core/src/it/units/sdm/project/game/gui/MainMenuScreen.java

```diff
@@ -2,23 +2,44 @@
 
 import com.badlogic.gdx.Gdx;
 import com.badlogic.gdx.Screen;
-import com.badlogic.gdx.graphics.OrthographicCamera;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.*;
 import com.badlogic.gdx.utils.ScreenUtils;
+import com.badlogic.gdx.utils.viewport.FitViewport;
 import org.jetbrains.annotations.NotNull;
 
 public class MainMenuScreen implements Screen {
-
-    private static final int VIEWPORT_WIDTH = 800;
-    private static final int VIEWPORT_HEIGHT = 400;
-    @NotNull
-    private final OrthographicCamera camera;
     @NotNull
     private final FreedomGame game;
+    @NotNull
+    private final Table initialMenu;
+    @NotNull
+    private final Skin skin;
+    @NotNull
+    private final Stage stage;
 
     public MainMenuScreen(final @NotNull FreedomGame game) {
         this.game = game;
-        camera = new OrthographicCamera();
-        camera.setToOrtho(false, VIEWPORT_WIDTH, VIEWPORT_HEIGHT);
+        stage = new Stage(new FitViewport(1200, 640), new SpriteBatch());
+        skin = new Skin(Gdx.files.internal("skin/uiskin.json"));
+        initialMenu = new Table();
+        initMenu();
+        stage.addActor(initialMenu);
+        Gdx.input.setInputProcessor(stage);
+    }
+
+    private void initMenu() {
+        initialMenu.setFillParent(true);
+        Image image = new Image(new Texture(Gdx.files.internal("freedom_logo.png")));
```

Comment:
We should dispose of textures that aren't used anymore. We'll have to generate an appropriate texture pack to use with an atlas at some point, but for now we should at least dispose of them in the `dispose()` method